ppr-ui/src/components/dialogs/SecuredPartyDialog.vue

```diff
@@ -69,8 +69,11 @@
                 party.personName.first+" "+party.personName.middle+" "+party.personName.last}}
               </div>
               <div :class="$style['addressText']">
-                {{ party.address.street }}, {{ party.address.city }}
-                {{ party.address.region }} , {{ party.address.postalCode }},
+                {{ party.address.street }},
+                {{ party.address.streetAdditional ? party.address.streetAdditional+"," : ""}}
```

Comment:
Super minor but a few options here! 
Perhaps spacing here `party.address.streetAdditional + ","`
OR
You can template string `${party.address.streetAdditional},`

Reply:
You will need back ticks wrapping your template string ` `

Reply:
fixed, thanks!